client/test/unitTests/Util/envUtil.test.ts

```diff
@@ -14,13 +14,7 @@ describe('envUtil', () => {
   const testLIB = '';
   const testAppURL = 'https://example.com';
   const testBasename = 'testBasename';
-  const testWorkbenchEndpoints = [
-    'one',
-    '/two',
-    'three/',
-    '/four/',
-    '/five/guy/',
-  ];
+  const testWorkbenchEndpoints = ['one', '/two', 'three/', '/four/'];
```

Comment:
Is there a reason to remove `/five/guy`/`

Reply:
We removed it because the array in this file was expecting five links including the terminal, so because we removed the terminal we had to remove one of the links

Reply:
Ok.
Just to make sure that longURLS are being parsed correctly, perhaps use this?
`const testWorkbenchEndpoints = ['one', '/two', 'three/', '/four/postfix'];`